fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -2237,6 +2240,11 @@ void ReadYourWritesTransaction::setOptionImpl(FDBTransactionOptions::Option opti
 		validateOptionValue(value, false);
 		options.specialKeySpaceChangeConfiguration = true;
 		break;
+	case FDBTransactionOptions::BYPASS_UNREADABLE:
+		validateOptionValue(value, false);
+		TraceEvent("ReadVersionStampValueOptionSet");
```

Comment:
More descriptive event name:
ReadVersionStampValueOptionBypassUnreadableSet

Reply:
Versionstamp is also usually treated as one word:

`ReadVersionstampBypassUnreadableSet`

Reply:
Is this something that we want logging on every transaction that uses it?